aries_cloudagent/core/dispatcher.py

```diff
@@ -284,9 +253,6 @@ def __init__(
 
         """
         super().__init__(**kwargs)
-        # Weakly hold the context so it can be properly garbage collected.
-        # Binding this DispatcherResponder into the context creates a circular
-        # reference.
```

Comment:
Is some auto-formatter removing these comments and newlines?

Reply:
@burdettadam -- please investigate this.  We definitely don't want to lose comments from the code.

Reply:
I absolutely agree, we don't want to lose any comments or clarity in code. I did notice some comments lost from some tooling I used. We are focused on getting all required interface updates done then updating tests. We will then go back over and add any lost comments and clarity. I also plan to squash my commits to make my git history clearer. 
Nice to see interest in the PR already. 😃.